code/game/machinery/doors/airlock_electronics.dm

```diff
@@ -5,6 +5,8 @@
 	var/list/accesses = list()
 	/// If the airlock should require ALL or only ONE of the listed accesses
 	var/one_access = 0
+	/// Checks to see if this airlock has an unrestricted helper (will set to TRUE if present).
+	var/unres_sensor = FALSE
 	/// Unrestricted sides, or sides of the airlock that will open regardless of access
 	var/unres_sides = 0
```

Comment:
same here

Reply:
I'm confused on what you mean. This is a boolean value, so what should I do here?